qa/L0_grpc_state_cleanup/test.sh

```diff
@@ -56,7 +56,7 @@ function check_state_release() {
   num_state_new=`cat $log_file | grep  "StateNew" | wc -l`
 
   if [ $num_state_release -ne $num_state_new ]; then
-    cat $log_file
+    # cat $log_file
```

Comment:
`log_file` is printed after the function call. Remove this line to avoid duplicate print.

Reply:
mu suggestion is to remove it then, i.e. delete line, not just comment it out

Reply:
Removed